src/tour.ts

```diff
@@ -136,31 +121,34 @@ tour.addStep({
   text: `This action will recolor all visible datasets. Keyboard Shortcut: r`,
   buttons: nextCancel,
 });
+
 tour.addStep({
   attachTo: {
     element: '[data-tour=fit]',
     on: 'auto',
   },
   title: 'Fit Data to Screen',
-  text: `This action will changes the chart view such that all selected datasets are fully shown. Keyboard Shortcut: f`,
+  text: `This action will change the chart view such that all selected datasets are fully shown. Keyboard Shortcut: f`,
```

Comment:
Do we want to remove this button now that we have the proper auto-fit mode?  Having two buttons that produce such similar behavior could be confusing.  Every use case i can think of for hitting this button would be achieved by hitting the auto-fit mode button instead.

If we get rid of this button, can we use its icon for the auto-fit mode button?